src/metrics/enhanced-metrics.ts

```diff
@@ -6,6 +6,7 @@ import { getColdStartTag } from "../utils/cold-start";
 import { getProcessVersion } from "../utils/process-version";
 import { writeMetricToStdout } from "./metric-log";
 
+import * as packageJson from "../../package.json";
```

Comment:
I'm going to approve for now. While I think this will probably work, we need to verify it with both the layer, and the npm package.